src/content/containers/ProfileViewerHeader.js

```diff
@@ -70,7 +70,7 @@ class ProfileViewerHeader extends PureComponent {
         }
       </div>
       <div className={`${className}HeaderIntervalMarkerOverviewContainer ${className}HeaderIntervalMarkerOverviewContainerGfx`}>
-        {
+        {/*
```

Comment:
I think we should keep the tracing markers at the top until this tab is more polished.